kv/memdb.go

```diff
@@ -0,0 +1,455 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Copyright 2015 Wenbin Xiao
```

Comment:
Who is this guy? @coocood

Reply:
he is our contributor https://github.com/pingcap/tidb/commits?author=xwb1989 :smile:

Reply:
He asked if he could put his name under PingCAP.
https://github.com/pingcap/tidb/pull/450#discussion_r43005838

Reply:
I have already rewrite the whole file🤣 I don't know if it is necessary to delete this line
BTW, do we need to update the license header (s/2015/2020/)?

Reply:
I think it is better to keep it as it used to be. I'm just curious about this, plz ignore me.